Mathlib/Data/Finset/Basic.lean

```diff
@@ -3847,6 +3856,35 @@ def sigmaEquivOptionOfInhabited (α : Type u) [Inhabited α] [DecidableEq α] :
           · simp }⟩
 #align equiv.sigma_equiv_option_of_inhabited Equiv.sigmaEquivOptionOfInhabited
 
+variable [DecidableEq α] {s t : Finset α}
+/-- `s ∪ t` (using finset union) is equivalent to `s ∪ t` (using set union) -/
+@[simps!]
+def finsetUnion (s t : Finset α) :
+    ((s ∪ t : Finset α) : Set α) ≃ (s ∪ t : Set α) :=
+  Equiv.Set.ofEq <| coe_union _ _
+
+/-- The disjoint union of finsets is a sum -/
```

Comment:
This is not fundamental, but sums are coproducts, hence I'd write this lemma as a map from s + t to s cup t.

Reply:
I indeed do use the symmetric version of these equivalences most of the time, so I've reversed them